packages/web/src/components/spa/Navigation/MobileNav/style.css

```diff
@@ -9,6 +9,31 @@
     color: white !important;
 }
 
+.navmobile-button-ham p {
+    padding: 0;
+    margin: 0;
+    font-size: xx-large;
+}
+
+.navmobile-button-ham {
```

Comment:
Have you tested the responsiveness of this button? I feel like there are too many fixed values which would break on different screens.